src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -780,6 +914,16 @@ public static class Sse2
         /// </summary>
         public static Vector128<double> Sqrt(Vector128<double> value) => Sqrt(value);
 
+        /// <summary>
+        /// __m128d _mm_sqrt_sd (__m128d a, __m128d b)
+        /// </summary>
+        public static Vector128<double> SqrtScalar(Vector128<double> a, Vector128<double> b) => SqrtScalar(a, b);
```

Comment:
I was thinking of calling `a` and `b`, `upper` and `value` respectively. Since `b` is the value we perform the operation on and `a` is the value we fill in the upper bits from. Thoughts?

Reply:
Do we really need to specifically fill the upper bits of the result in practice? 
For the performance perspective, we always recommend using the same register as the source and upper argument.
Especially, if we decide to support the two-parameter version of `Sqrt` intrinsic, on non-AVX machines, the compiler may have to insert `unpack` and `shuffle` instructions to implement this semantic, which they are both long latency instructions.

Reply:
In summary, I am suggesting that only expose the one-parameter intrinsic for SQRTSS and SQRTSD.

Reply:
Just exposing the single intrinsic version is probably fine. I actually missed that the two operand form is only on AVX and above, the Intel Intrinsics Guide lists it as SSE2: https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=Sqrt&techs=SSE,SSE2

Reply:
The Software Developers Manual lists the information correctly.